app/global.interfaces.ts

```diff
@@ -191,3 +191,18 @@ export interface Odds {
     Epic?: number
     Legendary?: number
 }
+
+
+//user needs to have some attributes as optional
+//player interface is an alternative to user with partial attributes
+// did not want to break app by changing user interface
+export interface Player {
```

Comment:
I'm torn on this one. Here are my thoughts:

- I like the lightweight version of the `User` interface
- The term `Player` seems too direct as this interface will only be used in rankings. However, in `useLeaderboardStore`, it's used to type an array of "players" so it makes sense. To me, `Player` would need to have more utility as a type within other parts of the code. 
- If `User` ever gets new attributes that we'd like to rank, we'd have to update both interfaces instead of one.

Think it over and get back to me. Essentially its just:
1. Use the User interface instead (tested this and does not seem to cause any issues).
2. Leave it as is. Not super important.
3. Rename the interface to be ranking specific.

up to you!